.github/actions/run-tests/action.yml

```diff
@@ -39,8 +39,8 @@ runs:
       shell: bash
       run: |
         if [[ "${{ inputs.check-python-bindings }}" != "true" ]]; then exit 0; fi
-        export PYTHONPATH="$PYTHONPATH:$(dirname $(find ${{ inputs.build-dir }}/install/ -name "pycvc5" -type d))"
-        python3 -c "import pycvc5"
+        export PYTHONPATH="$PYTHONPATH:$(dirname $(dirname $(find ${{ inputs.build-dir }}/install/lib -name 'cvc5_python_base*')))"
```

Comment:
On some platforms it might not install to `install/lib` but maybe `install/lib64`. I remember that Makai had issues like that, which was the reason for the `find type -d` approach.

Reply:
Thanks for catching this. The `/lib` restriction is indeed undesirable. See 882d481